public/app/features/alerting/components/AlertingQueryEditor.tsx

```diff
@@ -134,26 +178,33 @@ const addQuery = (
     model: {
       ...queryToAdd.model,
       hide: false,
-      refId: refId,
-    },
-    relativeTimeRange: {
-      from: 21600,
-      to: 0,
+      refId,
     },
+    relativeTimeRange: defaultTimeRange(queryToAdd.model),
   };
 
   return [...queries, query];
 };
 
+const defaultTimeRange = (model: DataQuery): RelativeTimeRange | undefined => {
+  if (isExpressionQuery(model)) {
+    return;
+  }
+
+  return {
+    from: 21600,
+    to: 0,
+  };
+};
+
 const getStyles = stylesFactory((theme: GrafanaTheme) => {
   return {
     container: css`
       background-color: ${theme.colors.panelBg};
       height: 100%;
     `,
-    refreshWrapper: css`
-      display: flex;
-      justify-content: flex-end;
+    runWrapper: css`
```

Comment:
We're moving all styles to GrafanaThemeV2 so maybe you should too?

Reply:
Yes!